### ISSUE TITLE
[FEATURE]  Добавить возможность одновременного открытия нескольких правил

### ISSUE THREAD

**Issue by dmtazetdinov** [Security-Experts-Community/vscode-xp#153]
### Is there an existing feature request or discussion for this?

- [X] I have searched the existing feature requests and discussions

### Problem

Требуется одновременно работать с несколькими корреляциями/локализациями/тестами, но при попытке открыть новое правило, например, чтобы скопировать небольшой кусок, то текущее окно заменяется те одновременно "просматривать" можно одно правило. Аналогично с тестами и локализациями, они не открываются в новых вкладках

### Solution

Сделать открытие всех корреляций/локализаций/тестов в новых вкладках

**Comment by DmitryOffsec** [comment]
Код всех правил открывается в любом количестве. Остальные вьюшки - да, в одном экземпляре.

**Comment by dmtazetdinov** [comment]
> Код всех правил открывается в любом количестве. Остальные вьюшки - да, в одном экземпляре.

Нет, только если ты что-то отредактируешь, а иначе в режиме чтения оно переоткрывается

**Comment by DmitryOffsec** [comment]
Не стоит путать функции расширения и самого VSCode. Это штатное поведение VSCode https://vscode.one/new-tab-vscode/

**Comment by dmtazetdinov** [comment]
> Не стоит путать функции расширения и самого VSCode. Это штатное поведение VSCode https://vscode.one/new-tab-vscode/

О, спасибо. Double-click the tab you want to keep open - а то напридумывают всякого
